asa_parser/src/asa_parse.py

```diff
@@ -57,4 +57,8 @@ def show_cpu_detailed(self):
 
     def ipsec_stats(self):
         """Parser for show ipsec stats"""
-        return json.dumps({'stats': self.get_show_section('ipsec stats')})
\ No newline at end of file
+        return json.dumps({'stats': self.get_show_section('ipsec stats')})
+    
+    def show_logging_buffered(self):
+        """Parser for show show logging buffered"""
+        return json.dumps({'text': self.get_show_section('show logging buffered')})
```

Comment:
the get_show_section() function automatically handles the work "show" - actually it's designed to handle custom prefixes too
```
    def get_section(self, sec_heading='show', sec_name=None):
```

Please change this to 
```
return json.dumps({'text': self.get_show_section('logging buffered')}
```
and the rest should work OK

Reply:
@LondonWesley any questions about the requested changes?